pkg/cmd/diagnostics/diagnostics_download.go

```diff
@@ -17,12 +17,21 @@ func newDiagnosticsDownloadCommand(ctx api.Context) *cobra.Command {
 		// the <bundle-id> is used to indicate what the expected positional arg is in the help output
 		Use:   "download <bundle-id>",
 		Short: "Download diagnostics bundle",
-		Args:  cobra.ExactArgs(1),
+		Args:  cobra.MaximumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			c := pluginutil.HTTPClient("")
 			client := diagnostics.NewClient(c)
 
-			id := args[0]
+			var id string
+			if len(args) == 0 {
+				bundle, err := latestBundle(client)
+				if err != nil {
+					return err
+				}
+				id = bundle.ID
+			} else {
```

Comment:
Don't need an else statement here

Reply:
But then we'd always try to use `args[0]` which would crash if there are no arguments given.